arch/arm64/boot/dts/qcom/sm7150-xiaomi-tucana.dts

```diff
@@ -138,6 +225,42 @@
 	enable-gpios = <&tlmm 67 GPIO_ACTIVE_HIGH>;
 };
 
+&panel {
+	compatible = "xiaomi,f4-41-06-0a";
+};
+
+&pm6150l_flash {
+	status = "okay";
+
+	/* Both LEDs */
+	led-1 {
+		function = LED_FUNCTION_FLASH;
+		color = <LED_COLOR_ID_WHITE>;
+		led-sources = <1 2>;
+		led-max-microamp = <300000>;
+		flash-max-microamp = <2000000>;
+		flash-max-timeout-us = <1280000>;
+	};
+	/* Only first LED */
+	led-2 {
+		function = LED_FUNCTION_FLASH;
+		color = <LED_COLOR_ID_WHITE>;
```

Comment:
Are you sure both LEDs are white? 

If this is not the case, please check out the list of all possible colors:
https://github.com/torvalds/linux/blob/master/include/dt-bindings/leds/common.h#L24